trivia.py

```diff
@@ -461,17 +484,24 @@ def _get_new_question(self):
         '''
         damaged_question = True
```

Comment:
Oh good lord I forgot I wrote this while loop. I'm pretty sure there's a better way to do this.